hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/metrics/RpcMetrics.java

```diff
@@ -147,6 +147,8 @@ public static RpcMetrics create(Server server, Configuration conf) {
   MutableCounterLong rpcRequeueCalls;
   @Metric("Number of successful RPC calls")
   MutableCounterLong rpcCallSuccesses;
+  @Metric("Number of observer namenode rejected RPC calls")
+  MutableCounterLong rpcRejectedByObserverCalls;
```

Comment:
I suggest renaming to `rpcCallsRejectedByObserver`

Reply:
Thanks sir suggestions, i will update it.